static/src/stylesheets/module/external/_witness.scss

```diff
@@ -69,54 +69,38 @@
     margin-bottom: 0;
 }
 
-.witness-cta-wrapper {
-    position: relative;
+.witness-cta {
     overflow: hidden;
-    height: 47px;
     margin-bottom: ($gs-baseline/3)*4;
 }
-.witness-cta__body {
-    @include fs-textSans(3);
-    float: left;
-    line-height: 15px;
-    padding: $gs-baseline/2 $gs-gutter/2 0;
-    margin-left: 50px;
 
-    .witness-cta__logo {
-        display: block;
-    }
+.button--witness {
```

Comment:
From the logic of the BEM naming convention I would look for the '.button--witness' modification in the 'button.scss' file.

Ideal solution would be to use some generic button modification we are already using on the page ('.button--large') so we wouldn't be creating another version of the button to the code base. But this is going down to UX and design.

If we want to create some specific version of the button just for some unique object ('.witness' object in this case) I would go for something like '.witness__button'. From this it makes perfect sense that button is using object '.button' but also specific element styles of the object '.witness'.